akka-stream/src/main/scala/akka/stream/impl/RetryFlowCoordinator.scala

```diff
@@ -54,9 +54,10 @@ import akka.util.OptionVal
     private var elementInProgress: OptionVal[In] = OptionVal.none
     private var retryNo = 0
 
-    setHandler(
+    setHandlers(
       externalIn,
-      new InHandler {
+      internalOut,
+      new InHandler with OutHandler {
```

Comment:
shouldn't that even be on the `TimerGraphStageLogic(shape) extends InHandler with OutHandler`?